builders/ethereum/dev-env/hardhat.md

```diff
@@ -83,7 +99,55 @@ Next, you'll need to modify your configuration file to add the network configura
 - `chainId` - the chain ID, which is used to validate the network
 - `accounts` - the accounts that can be used to deploy and interact with contracts. You can either enter an array of the private keys for your accounts or use an [HD Wallet](https://github.com/ethereumbook/ethereumbook/blob/develop/05wallets.asciidoc#hierarchical-deterministic-wallets-bip-32bip-44){target=\_blank}
 
-For this example, the network will be Moonbase Alpha, but you can modify the configuration to use any of the Moonbeam networks:
+Hardhat 3 includes an encrypted secrets manager that makes it easier to handle sensitive information like private keys. This ensures you don't have to hardcode secrets in your source code or store them in plain text.
+
+!!! note
+    The encrypted secrets manager is only available in Hardhat 3 or higher. As of writing this guide, Hardhat 3 is in alpha. You can install the latest alpha version with:
+
+    ```bash
+    npm install hardhat@3.0.0-next.5
+    ```
+
+    For the latest releases and updates, check the [Hardhat releases page](https://github.com/NomicFoundation/hardhat/releases/).
+
+To use encrypted secrets, you'll need to:
+
+1. Install Hardhat 3 or later:
+```bash
+npm install --save-dev hardhat@^3.0.0
```

Comment:
This command did not work for me. It returned an error :

"npm error notarget No matching version found for hardhat@^3.0.0.
npm error notarget In most cases you or one of your dependencies are requesting
npm error notarget a package version that doesn't exist."

If I run the command in your note right above this, it works with no error.

Reply:
Thx fixed!